lib/elixir/lib/enum.ex

```diff
@@ -1234,21 +1234,27 @@ defmodule Enum do
   Returns a list of results of invoking `fun` on every `nth`
   item of `enumerable`, starting with the first element.
 
-  The first item is always passed to the given function.
+  The first item is always passed to the given function, unless `nth` is `0`.
 
   The second argument specifying every `nth` item must be a non-negative
   integer.
 
+  If `nth` is `0`, then `enumerable` is diretcly converted to a list,
+  witouth `fun` being ever applied.
+
   ## Examples
 
-      iex> Enum.map_every(1..10, 2, fn(x) -> x * 2 end)
-      [2, 2, 6, 4, 10, 6, 14, 8, 18, 10]
+      iex> Enum.map_every(1..10, 2, fn(x) -> x + 1000 end)
```

Comment:
Since you're touching these lines, could you remove the parens in `fn(x)` and make it `fn x`?